client/client.js

```diff
@@ -35,6 +66,11 @@ function Client(apiUrl) {
   // Populates the cache of models so you can access them with
   // client.models.ModelName.
   client.models();
+
+  // Override and implement the remote methods that use HTTP context because
+  // LB will not provide the correct client implementation.
```

Comment:
Its a subtle point, hard to understand concisely, maybe insert the above comment into the head of one of the implementation files?